### PR TITLE
Service member hub

### PR DESCRIPTION
**Pull request by DanielleSoCompany**
## Pages edited

**Existing pages edited in this PR:**
/health-care/eligibility/
/life-insurance/options-eligibility/sgli/

**New pages in this PR:**
/service-member-benefits
/health-care/eligibility/active-duty

**Legacy urls deleted as part of this PR:**
/health-care/eligibility/returning-servicemembers
/health-care/eligibility/active-duty-servicemembers

## Origin of request (internal/stakeholder/user feedback)
new service member hub

## Description of what's needed (edits/link changes/additions)
creating new hub and creating/editing relevant pages

### CODE REVIEW

**Comment by DanielleSoCompany** [comment]
**NOTE** Please do not merge this PR.

File: pages/health-care/eligibility/active-duty.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+Call our health benefits hotline at 1-877-222-VETS (<a href="tel:+18778387">1-877-8387</a>) to connect with a benefits counselor.  If you have hearing loss, call TTY: 1-800-877-8339. We're Monday through Friday, 8:00 a.m. to 8:00 p.m. ET. Or contact your nearest VA medical center. <br>
+[Find a VA medical center near you](/find-locations/?zoomLevel=4&page=1&address=&facilityType=health&serviceType)
+
+<script type="text/javascript">
+  // Toggle the expandable crisis info
+  document.getElementById('crisis-expander-link')
+    .addEventListener('click', function () {
+      document.getElementById('crisis-expander-content').classList.toggle('expander-content-closed');
+    });
+</script>
```
Comment (ncksllvn) [inline pages/health-care/eligibility/active-duty.md]: This script is used to expand content for Crisis Line alerts, but needs to be updated for the Mental Health alerts. Would you like me to update this?

Comment (DanielleSoCompany) [reply pages/health-care/eligibility/active-duty.md]: Yes, please - thank you!

**Comment by DanielleSoCompany** [comment]
@kevwalsh @beeyayjay (cc @jenniferlee-dsva) We will not have SME approval on content in time to merge the health care page in this PR in time for the content freeze. So I've pulled it from this PR and we'll plan to do a manual migration once the content is fully approved. I'm removing the MVP content label.

**Pull request merged**